R/g.getstarttime.R

```diff
@@ -47,7 +47,7 @@ g.getstarttime = function(datafile, P, header, mon, dformat, desiredtz, configtz
     starttime = as.POSIXlt(starttime)
   } else if (dformat == 2 & (mon == 3 | mon == 4 | mon == 6)) {
     if (mon == 3 | mon == 6) {
-      tmph = read.csv(datafile, nrow = 8, skip = 1)
+      tmph = data.table::fread(datafile, nrows = 8, skip = 1, data.table = FALSE)
       tmphi = 1
```

Comment:
Please undo these change to g.getstarttime, as we discussed [here](https://github.com/wadpac/GGIR/pull/791#issuecomment-1566764332). It has taken a lot of time to develop and test the code for reading csv files. Changing it without addressing a clear bug and without a clear functional improvement seems like an unnecessary risk.